core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1873,11 +1865,13 @@ id < (
 		ORDER BY id DESC
 		LIMIT $3
 	) numbers
-)`, subject, subject, queueSize)
+) RETURNING id`, subject, subject, queueSize)
 		if err != nil {
+			if errors.Is(err, sql.ErrNoRows) {
+				return nil
+			}
 			return pkgerrors.Wrap(err, "DeleteUnstartedEthTx failed")
```

Comment:
nit: Since you're already updating this method, could you also move this to using `fmt.Errorf`
```suggestion
			return fmt.Errorf("DeleteUnstartedEthTx failed: %w", err)
```